src/mlpack/tests/lsh_test.cpp

```diff
@@ -9,103 +9,211 @@
 #include "old_boost_test_definitions.hpp"
 
 #include <mlpack/methods/lsh/lsh_search.hpp>
+#include <mlpack/methods/neighbor_search/neighbor_search.hpp>
 
 using namespace std;
 using namespace mlpack;
 using namespace mlpack::neighbor;
 
+double compute_recall(
+    arma::Mat<size_t> LSHneighbors, 
+    arma::Mat<size_t> groundTruth)
+{
+  const int n_queries = LSHneighbors.n_cols;
+  const int n_neigh = LSHneighbors.n_rows;
```

Comment:
Really minor style note: we should avoid underscores in variable names... I'd suggest just `queries` and `neigh`.